CONFIG.md

```diff
@@ -3,9 +3,9 @@
 ## Endpoints
 The most important things that can be customized are endpoints about Keystone and Keypass.
 Some specific operations involve calls to others endpoints like Orion, Iota or Perseo.
-And there are others endpoints that are not invoked but are keept to use into Orion subscriptions like STH, Perseo and Cygnus.
-
+And there are others endpoints that are not invoked but are keept to use into Orion subscriptions like STH, Perseo, Cygnus and LDAP.
```

Comment:
Typo: keept -> kept

Reply:
fixed in https://github.com/telefonicaid/orchestrator/pull/121/commits/6edd6316796e2d5e43d7fff7e4b1d1227335bb45